devtools/dump_devinfo.py

```diff
@@ -140,7 +147,17 @@ async def handle_device(
         ]
 
     for fixture_result in fixture_results:
-        save_filename = Path(basedir) / fixture_result.folder / fixture_result.filename
+        save_folder = Path(basedir) / fixture_result.folder
+        if save_folder.exists():
+            save_filename = save_folder / f"{fixture_result.filename}.json"
+        else:
+            # If being run without git clone
+            save_folder = Path(basedir) / NO_GIT_FIXTURE_FOLDER
```

Comment:
Perhaps it would be better to prompt for the location, to make it very clear to the user where the results are being stored?

Reply:
There is a `basedir` option to the cli and the `save y/n` message to the user includes the location, so I think a prompt is probably unnecessary.

Reply:
Ok, ready to go then!